values-schema.yaml

```diff
@@ -377,6 +405,22 @@ definitions:
       id:
         $ref: '#/definitions/idName'
         description: Must be the same as the name.
+      jobs:
+        description: |
+          You may pass an array of Job objects. The Kubernetes API accepts a Job specification, 
+          which is meant for run-to-completion apps (e.g. batch Jobs).
+
+              enabled: (type boolean)
+                  env: (type strings[], pattern-matched)
+                  image: (type string, pattern-matched)
+                  init: (type object)
+                    image: (type string, pattern-matched)
+                    script: (type string)
+                    secret: (type string[])
```

Comment:
Please refrain from these using such blocks and let prop descriptions speak for themselves. This is not documentation.

Reply:
I realize that, I'll delete it in a sec. But maybe you thought this feature would be convenient for a client who is writing a new yaml file in otomi-values. It would look like this:


![](https://user-images.githubusercontent.com/20700197/101772400-9cf59d80-3aeb-11eb-975b-580fd8cf0df5.png)

![](https://user-images.githubusercontent.com/20700197/101772435-a67f0580-3aeb-11eb-82f7-9c661c7518f2.png)

Reply:
It's what most extensions use of, that's how I got the idea. So for example with the Kubernetes extension you can do something like this:

<img width="702" alt="Screenshot 2020-12-11 at 10 04 26" src="https://user-images.githubusercontent.com/20700197/101884019-5ad66580-3b98-11eb-9ed2-5efeaae61c5d.png">

<img width="472" alt="Screenshot 2020-12-11 at 10 05 21" src="https://user-images.githubusercontent.com/20700197/101884054-69248180-3b98-11eb-92fc-f3ab8aecb05d.png">
<img width="702" alt="Screenshot 2020-12-11 at 10 04 26" src="https://user-images.githubusercontent.com/20700197/101884070-6fb2f900-3b98-11eb-8eb3-c8b16077a8d5.png">

![](https://user-images.githubusercontent.com/20700197/101772435-a67f0580-3aeb-11eb-82f7-9c661c7518f2.png)

Reply:
Maybe it's a bit overkill for now.